members_only/js/components/pages/Setup.js

```diff
@@ -1,4 +1,5 @@
 import React, {Component} from 'react';
+import StripeCheckout from 'react-stripe-checkout';
```

Comment:
This dependency is not in package.json. Please commit it.

Reply:
Done